.travis.yml

```diff
@@ -188,16 +189,6 @@ matrix:
                INSTALL_WITH_PIP=True
                EXTRAS_INSTALL="docs"
 
-        # Testing with Travis provided Python3.8. Change this job to a regular
-        # one once conda supports python3.8. Make sure we use released numpy rather
-        # than some kind of dev coming natively from the Travis environment
-        - language: python
-          stage: Final tests
-          dist: xenial
-          python: 3.8-dev
-          env: INSTALL_WITH_PIP=True EXTRAS_INSTALL="test" USE_CI_HELPERS=False
-               OTHER_DEPENDENCY_PREFERENCES='numpy==1.17.3' PIP_FLAGS="--pre"
-
```

Comment:
Does Travis not provide ``3.9-dev`` yet? If it does, we should keep this section and use that?

Reply:
What about using ``nightly``?

Reply:
Is there value in testing against Python dev?

Reply:
Well I was thinking to pick up early issues, but it might be quite an unstable build. So feel free to ignore. It would be nice if there was an easy way to automatically test against actual Python pre-releases though if available but not sure if there's an easy way to do that.

Reply:
python pre releases doesn't come that frequently, I think we're fine by adding tests against them consciously.  
In the past we had issues to fix for each new python version anyway, it couldn't have been a seamless addition.  So I would vote against adding a dev build, it would just create failing CI that may need the attention of someone following the python dev, and I would rather have that as a regular task sometimes in the middle of the dev cycle, than a sudden situation due to a non regular contributor runs into unrelated CI issues that need either explanation or fixing.

Reply:
So, should I close #9201 as won't fix?

Reply:
Am I contradicting myself? what an unusual situation ;)

But, given that #9201  is about a cron job, doing this as e.g a weekly cron might make sense. 🤷‍♀

Reply:
(oh, and to answer the question above, travis doesn't yet do 3.9-dev, but nightly)

https://docs.travis-ci.com/user/languages/python/#specifying-python-versions